id/list.json

```diff
@@ -0,0 +1,34 @@
+{
+	"data": [{
+			"year": 2018,
+			"url": "2018/",
+			"seasons": [{
+					"season": "spring",
+					"title": "spring",
```

Comment:
for each "title" attribute, you should translate the English version to the local language (Indonesian in you case)